koji/collection-mash-split.py

```diff
@@ -440,9 +491,14 @@ def main():
         el8_candlepin = MashConfig(collection, version, "candlepin-el8", "candlepin-el8", "candlepin/el8")
         el8_candlepin.name = 'katello-candlepin-{}-el8'.format(version)
```

Comment:
This almost makes me think candlepin deserves is own module metadata. Probably overkill now, but it is a separate repo.

Reply:
I would rather question why it is a separate repo…

Reply:
I'm guessing partly historical. Another is that Candlepin is tagged in Koji, rather than that the source lives in foreman-packaging.